src/main/scala/ai/eto/rikai/sql/model/Registry.scala

```diff
@@ -42,9 +44,8 @@ private[rikai] object Registry {
 
   val REGISTRY_IMPL_PREFIX = "rikai.sql.ml.registry."
   val REGISTRY_IMPL_SUFFIX = ".impl"
-  val REGISTRY_IMPL_DEFAULT_KEY = "rikai.sql.ml.registry.impl"
+  val DEFAULT_URI_ROOT_KEY = "rikai.sql.ml.registry.uri.root"
```

Comment:
mind to write some doc here THAT IS not only scheme?

`uri.root` is not an concept w/o confusions.

Reply:
fair. will add documentation. but first, sleep

Reply:
ok docs updated.